packages/zcli-apps/src/commands/apps/update.ts

```diff
@@ -35,21 +35,30 @@ export default class Update extends Command {
     try {
       const { app_id }: any = await getUploadJobStatus(job_id, appPath)
       cli.action.stop('Deployed')
+      const updatedSummary: string[] = []
```

Comment:
nit: I guess we can extract the following code out to be a function and only return a boolean if you need. Then you should be early return in the Object.keys.forEach easily. And you don't need to concat the string and filter the string at the end.

Reply:
Addressed by https://github.com/zendesk/zcli/pull/75/commits/1fc8dc0ffde660ced67bbf6af8a20e7eb0a1d99e